doc/source/geotransformations.md

```diff
@@ -0,0 +1,204 @@
+```
+
+As shown above, if the rasters have different {ref}`Pixel interpretation<pixel-interpretation>`, GeoUtils 
+raises a warning to ensure this is intended. 
+This warning can be turned off at the package-level using [GeoUtils' global configuration]().
+
+```{code-cell} ipython3
+:tags: [hide-input]
+:mystnb:
+:  code_prompt_show: "Show the code for plotting the figure"
+:  code_prompt_hide: "Hide the code for plotting the figure"
+
+f, ax = plt.subplots(1, 3)
+ax[0].set_title("Raster 1")
+rast.plot(ax=ax[0])
+vect.plot(rast, ax=ax[0], ec="k", fc="none")
+ax[1].set_title("Raster 2")
+rast2.plot(ax=ax[1], cmap="Reds",)
+vect.plot(rast, ax=ax[1], ec="k", fc="none")
+ax[2].set_title("Match-ref\nreprojection")
+rast_reproj2.plot(ax=ax[2])
+vect_reproj.plot(ax=ax[2], ec="k", fc="none")
+_ = ax[1].set_yticklabels([])
+_ = ax[2].set_yticklabels([])
+```
+
+## Crop or pad
+
+{func}`geoutils.Raster.crop` or {func}`geoutils.Vector.crop`.
+
+Cropping **modifies the spatial bounds of the geospatial data in a rectangular extent**, by removing or adding data 
+(in which case it corresponds to padding) without resampling. For this, we use {func}`~geoutils.Raster.crop`.
+
+For rasters, cropping removes or adds pixels to the sides of the raster grid.
+
+For vectors, cropping removes some geometry features around the bounds, with three options possible:
+1. Removing all features **not intersecting** the cropping geometry,
+2. Removing all features **not contained** in the cropping geometry,
+3. Making all features **exactly clipped** to the cropping geometry (modifies the geometry data).
+
+```{code-cell} ipython3
+# Clip the vector to the raster
+vect_clipped = vect_reproj.crop(rast, clip=True)
+```
+
+```{code-cell} ipython3
+:tags: [hide-input]
+:mystnb:
+:  code_prompt_show: "Show the code for plotting the figure"
+:  code_prompt_hide: "Hide the code for plotting the figure"
+
+f, ax = plt.subplots(1, 2)
+ax[0].set_title("Before clipping")
+rast.plot(ax=ax[0])
+vect_reproj.plot(ax=ax[0], ec="k", fc="none")
+ax[1].set_title("After clipping")
+rast.plot(ax=ax[1])
+vect_clipped.plot(ax=ax[1], ec="k", fc="none")
+```
+
+## Shift
+
+{func}`geoutils.Raster.shift` or {func}`geoutils.Vector.shift`.
+
+Shifting **modifies the georeferencing of the data by a horizontal offset** without modifying the underlying data,
+which is especially useful to align the data due to positioning errors. For this, we use {func}`~geoutils.Raster.shift`.
+
+
+```{code-cell} ipython3
+# Shift the raster by a certain offset
+rast_shift = rast.shift(1000, 1000)
+```
+
+```{code-cell} ipython3
+:tags: [hide-input]
+:mystnb:
+:  code_prompt_show: "Show the code for plotting the figure"
+:  code_prompt_hide: "Hide the code for plotting the figure"
+
+f, ax = plt.subplots(1, 2)
+ax[0].set_title("Before shifting")
+rast.plot(ax=ax[0])
+vect_clipped.plot(ax=ax[0], ec="k", fc="none")
+ax[1].set_title("After shifting")
+rast_shift.plot(ax=ax[1])
+vect_clipped.plot(ax=ax[1], ec="k", fc="none")
```

Comment:
Might want to add a `plt.tight_layout` here as the subplots are overlapping. I was also thinking yesterday. We should improve the x/y ticks when the values are very large... But probably not an easy task.
<img width="812" alt="image" src="https://github.com/GlacioHack/geoutils/assets/3285905/c44d095b-0866-45a9-8cdd-34d00784cd62">

Reply:
Done